appendix/how-to-read-a-network-map.md

```diff
@@ -6,7 +6,7 @@ Everything that we build in this project depends on us understanding the shape o
 
 Let's start with a very simple network map:
 
-![basic-network-map](img/basic-network-map.svg)
+![basic-network-map](../miscellaneous/basic-network-map.svg)
```

Comment:
In lieu of a miscellaneous dir, have you considered having a diagrams/ dir instead?  maybe labeled by chapter (but I can see arguments for not doing that, e.g. when referenced over multiple chapters).

Reply:
I'm not sure what was wrong with using the existing `img` directory?

Reply:
Ah, good thinking!